index.html

```diff
@@ -108,6 +108,34 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <div class="card">
```

Comment:
Please keep the comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` and change it to
into `<!-- ________ Your_Name Contributor card START ________ -->`

Reply:
Got it!